src/webui/api/appcontroller.cpp

```diff
@@ -78,14 +78,26 @@ void AppController::versionAction()
 
 void AppController::buildInfoAction()
 {
+    const auto platformName = []() -> QString
+    {
+#ifdef Q_OS_MACOS
+        return u"macos"_s;
+#elif defined(Q_OS_WIN)
+        return u"windows"_s;
+#else
+        return u"linux"_s;
+#endif
```

Comment:
Is this some kind of internal data or is it visible to the user?

Reply:
It is just some defined values for platform identification so it is more like internal data.